src/ol/PluggableMap.js

```diff
@@ -1031,7 +1031,7 @@ class PluggableMap extends BaseObject {
 
       if (!this.handleResize_) {
         this.handleResize_ = this.updateSize.bind(this);
-        addEventListener(EventType.RESIZE, this.handleResize_, false);
+        window.addEventListener(EventType.RESIZE, this.handleResize_, false);
```

Comment:
Would `global` instead of `window` also work?

Reply:
yes that should also do the trick, but we had other occurences of window in the source so I went with that

Reply:
I verified that global also works, see #9601 for the update